static/admin/config.yml

```diff
@@ -116,6 +116,22 @@ collections:
             default: 0
             valueType: float
             widget: nonEmptyStringableNumber
+      - label: "Linked Article"
```

Comment:
Why are these medium posts attached to a project? I don't think that's necessarily always the case. We could have articles that are unrelated to any project, but still want to highlight them on the homepage.

I would suggest looking at adding this to the list of projects as defined within the `index` page segment in config.yml. Or alternatively a separate part of the `index` page where you would define the block and at which point in the list it would appear (maybe by entering a number).

I think in general we would only have a single block pointing to medium in between the projects on the homepage, and it won't be necessarily related to any project.

Reply:
.

Reply:
Have created a stand alone type of `articles` now instead. Can be references(relation) in the projects list in the homepage CMS and ordered to liking

https://github.com/RandomStudio/random-studio-gatsby/pull/70/files/d5269b8afd6669d9156be270b4cb258c5f2b805f..085639c34d47ba136661cbb98893864efbaae4e5

Reply:
It's a big improvement, and I kind of like the separate article type. On the other hand, it is still very much connected to a project when you add it to the homepage. I think that is still not semantically correct. I would prefer a way of being able to add an article link to the list of homepage items without having to attach it to a project, although I'm not sure what the best way to do that in Netlify CMS would be. Maybe you have some smart idea?

Reply:
@larixk 
You mean to have two separate lists (pseudo thoughts) - projects & articles and have some sort of control over the position of the article orders?

Reply:
Have detached the articles and added a extra field called `position` to the linked article relation.
In code it will position the article on given position (index + 1 based)